src/matrix/matrixnormal.jl

```diff
@@ -0,0 +1,159 @@
+"""
+    MatrixNormal{T <: Real, ST <: AbstractPDMat} <: ContinuousMatrixDistribution
+
+Matrix Normal Distribution
+
+`X ~ MN(M, U, V)`
+
+M: n x p
+
+U: n x n positive definite
+
+V: p x p positive definite
+
+f(X) = c0 * exp( -0.5 tr[inv(V) (X - M)' inv(U) (X - M)] )
+
+c0   = (2pi) ^ {-np / 2} |V| ^ {-n / 2} |U| ^ {-p / 2}
+
+https://en.wikipedia.org/wiki/Matrix_normal_distribution
+
+"""
+struct MatrixNormal{T <: Real, ST <: AbstractPDMat} <: ContinuousMatrixDistribution
+
+    M::AbstractMatrix{T}
```

Comment:
This should probably now be something like
```julia
struct MatrixNormal{T <: Real, TM <: AbstractMatrix, ST <: AbstractPDMat} <: ContinuousMatrixDistribution
    M::TM
    U::ST
    V::ST
    c0::T
end
```
Having an abstract type as a parameter here will cause type stability issues.

Reply:
Changed.